pipeline/transformer/transformer_miro/src/main/scala/uk/ac/wellcome/platform/transformer/miro/transformers/MiroImage.scala

```diff
@@ -26,7 +19,7 @@ trait MiroImage {
     imageUriTemplate.format(iiifImageApiBaseUri, miroId)
   }
 
-  def getImage(miroRecord: MiroRecord): UnmergedImage[Unminted] =
+  def getImage(miroRecord: MiroRecord): UnmergedImage[Identifiable] =
```

Comment:
mmmh I don't think I follow.. 
If something is specified as `Identifiable`, in what way would `Unidentifiable` seem possible?

I think there was some discussion back then about whether `Identifiable` should be ` Unidentified` but `Identifiable` was chosen because `Unidentified` is not specific about the fact that the object is not identified _yet_ but it definitely will be.

Reply:
> Maybe Identifiable too doesn't make it too clear that the object will have an identifier

I think that's it, for me at least.

> If something is specified as Identifiable, in what way would Unidentifiable seem possible?

I know this isn't really possible in our types but it's just confusing to me, I think for the reason above

Reply:
I see. Yeah it is a bit obscure. I think we never managed to find names that we were completely happy with. 
* `Identifiable` means "has sourceIdentifier so will have canonicalId"
* `Unidentifiable` means "doesn't have a sourceIdentifier so it will never have a canonicalId"
* `Identified` means "has a sourceIdentifier AND a canonicalId"
I never managed to find names that quite reflected the above.